### ISSUE TITLE
All 1.6 commands fail with SyntaxError: Unexpected end of JSON input

### ISSUE THREAD

**Issue by noahsw** [meteor/meteor#9395]
I first thought this was specific to my app (https://github.com/meteor/meteor/issues/9348) but I get the same error when simply running `meteor help`.

```
ubuntu@box2519:~/.meteor$ cat /etc/os-release
NAME="Ubuntu"
VERSION="14.04.4 LTS, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.04.4 LTS"
VERSION_ID="14.04"

ubuntu@box2519:~/.meteor$ uname -a
Linux box2519.localdomain 3.13.0-129-generic #178-Ubuntu SMP Fri Aug 11 12:48:20 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

ubuntu@box2519:~/.meteor$ which meteor
/usr/local/bin/meteor

ubuntu@box2519:/usr/local/bin$ meteor npm -v
5.4.2

ubuntu@box2519:/usr/local/bin$ meteor node -v
v8.8.1

ubuntu@box2519:/usr/local/bin$ meteor help
undefined:1
t


SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Object.<anonymous> (/tools/fs/safe-watcher.js:15:30)
    at Module._compile (module.js:612:30)
    at Object.Module._extensions..js (module.js:623:10)
    at Module.load (module.js:531:32)
    at Module.Mp.load (/tools/tool-env/install-runtime.js:16:29)
    at tryModuleLoad (module.js:494:12)
    at Function.Module._load (module.js:486:3)
    at Module.require (module.js:556:17)
    at require (internal/module.js:11:18)

ubuntu@box2519:/usr/local/bin$ meteor show meteor
undefined:1
t


SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Object.<anonymous> (/tools/fs/safe-watcher.js:15:30)
    at Module._compile (module.js:612:30)
    at Object.Module._extensions..js (module.js:623:10)
    at Module.load (module.js:531:32)
    at Module.Mp.load (/tools/tool-env/install-runtime.js:16:29)
    at tryModuleLoad (module.js:494:12)
    at Function.Module._load (module.js:486:3)
    at Module.require (module.js:556:17)
    at require (internal/module.js:11:18)
```

**Comment by abernix** [comment]
What do you have `METEOR_WATCH_FORCE_POLLING` set to?

    echo $METEOR_WATCH_FORCE_POLLING

**Comment by noahsw** [comment]
```
environment versions:
operating system 14.04.4 LTS, Trusty Tahr
kernel Linux box343 3.13.0-129-generic #178-Ubuntu SMP Fri Aug 11 12:48:20 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
which node: /opt/circleci/nodejs/v8.8.1/bin/node
node v8.8.1
METEOR_WATCH_FORCE_POLLING: t
meteor node v8.8.1
yarn 1.1.0
Google Chrome 62.0.3202.94 
```

Weird. Somehow it has the value of "t" ? No idea what is setting that but that would explain the
```
undefined:1
t
```

**Comment by noahsw** [comment]
Ah, I set it based on https://github.com/meteor/docs/blob/version-NEXT/long-form/file-change-watcher-efficiency.md

```
To improve this situation, on the machine where you are running meteor (eg, inside your VM), set the METEOR_WATCH_FORCE_POLLING environment variable to any non-empty value. For example, put the line *METEOR_WATCH_FORCE_POLLING=t* into the .bashrc file in your home directory.
```

**Comment by noahsw** [comment]
Removing that environment variable fixed things! https://circleci.com/gh/winsleague/winsleague/1029

**Comment by abernix** [comment]
Thanks for reporting back!  Yeah, it used to be somewhat common practice to just set the environment variables to _something_, where the value was not necessary, only the variable's existence. I guess `t` (for `true`) was short and sweet!

We've changed this behavior on some variables on more recent versions in order to allow negation of these settings (via `VAR=false`, for example), especially for variables which default to `true`.  The value in this case is now parsed by `JSON.parse`. 

I've updated that document, even though it's not really actively maintained anymore.  (Those docs are archived "long-form" explanations which were extracted from the now-defunct GitHub Wiki pages.)
